buildspec.yml

```diff
@@ -10,8 +10,10 @@ phases:
   build:
     commands:
       - ${CODEBUILD_SRC_DIR}/get-env.sh docker_serverless_build
-      - docker build ${CODEBUILD_SRC_DIR} --tag ${IMAGE_REPO_URI}:${COMMIT_HASH} --tag ${IMAGE_REPO_URI}:latest
```

Comment:
(line 8) Seems like we should be using the latest ECR login technique to deal with multiple versions of AWS CLI.

Reply:
fixed